test/benchmark/cli.ts

```diff
@@ -132,18 +148,53 @@ $ node test/benchmark/cli -n 20
             .map((dirEntry) => dirEntry.name.replace(/\.js$/, ''));
 
         for (const benchmark of benchmarks) {
-            runCommands.push(
-                `${packageName}:${benchmark}=test/benchmark/bench-runner.html` +
-                    `?bench=${benchmark}` +
-                    `&package=${packageName}`
-            );
+            if (!config.benchmarks) return;
+            config.benchmarks.push({
```

Comment:
In regards to the unpublished npm package issue I ran into with field-group, it would be a nice-to-have in the future to skip the remote test if its not available.

a couple options might be to either to use some standardized proess with the package version to determine if its been published or to actually check for latest in npm (although this might be a bit more heavy)
```
yarn info @spectrum-web-components/field-group@latest
```

speaking of which, it might also make sense to limit the benchmark tests to the same major version instead of against latest, in case theres been an interface change.

Reply:
Great idea! I've added the following message when a package is still on the "default" (read here as "unpublished) version:
```
⚠️  It looks like '${packageName}' has yet to be published to NPM. Skipping!
```
I've also added the `-c/--compare` flag which allows for the use of a specific version of `@spectrum-web-components/bundle` to be used as the "remote" source against which to test. This is done so that packages that require child components both public or private are able to rely of releases of all the comparable version at a time.